helpers/config.js

```diff
@@ -5,11 +5,16 @@ module.exports = {
 function getBootstrapConfig() {
   return {
     authRequired: strToBool(process.env.AUTH_REQUIRED, true),
-    cors: { headers: (process.env.CORS_HEADERS || '').split(',').filter(Boolean) },
+    cors: { headers: strToArray('CORS_HEADERS').filter(Boolean) },
```

Comment:
let's also include filter(Boolean) in the strToArray, as it removes empty elements for strings like `''` or `foo,bar,`